vllm/engine/metrics.py

```diff
@@ -61,6 +61,10 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             labelnames=labelnames)
 
         # Iteration stats
+        self.counter_num_preemption = Counter(
+            name="vllm:num_preemption",
```

Comment:
this should be `vllm:num_preemption_total` to match prometheus naming best practices

https://www.google.com/search?q=prometheus+naming+best+practices&oq=prometheus+naming+best+practices&gs_lcrp=EgZjaHJvbWUyBggAEEUYOdIBCDM3MTFqMGo0qAIAsAIB&sourceid=chrome&ie=UTF-8

(side note:`vllm:` is not best practices, should be `vllm_` but we have already committed the sin)